src/library.cpp

```diff
@@ -676,7 +673,8 @@ void lammps_extract_box(void *handle, double *boxlo, double *boxhi,
 
 This function sets the simulation box dimensions (upper and lower bounds
 and tilt factors) from the provided data and then re-initializes the box
-information and all derived settings.
+information and all derived settings. It may only be called before atoms
```

Comment:
@sjplimp here is the requested change.

Reply:
yes, looks good - thanks

Reply:
@akohlmey Thanks, this looks good and seems to be working.